go-controller/pkg/ovn/egressip.go

```diff
@@ -1835,6 +1835,8 @@ type egressIPController struct {
 	nbClient libovsdbclient.Client
 	// watchFactory watching k8s objects
 	watchFactory *factory.WatchFactory
+	// EgressIP Node reachability check configuration
+	egressIPNodeReachConfig config.EgressIPConfig
```

Comment:
Feels like this should really be called `egressIPConfig` is we're going with a dedicated struct called `config.EgressIPConfig`...I think we should just add this part to `OVNKubernetesFeatureConfig` however.